OrbitGl/TracepointInfoManager.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_TRACEPOINT_INFO_MANAGER_H_
+#define ORBIT_TRACEPOINT_INFO_MANAGER_H_
+
+#include <optional>
+#include <string>
+
+#include "BlockChain.h"
+#include "absl/container/flat_hash_map.h"
+#include "absl/synchronization/mutex.h"
+#include "capture_data.pb.h"
+#include "tracepoint.pb.h"
+
+class TracepointInfoManager {
```

Comment:
As briefly discussed, this should be created outside OrbitGl so the client ggp can make use of it, as it is independent from OrbitGl. Thanks!

Reply:
Thank you so much for pointing that out!